lib/snowflex/transport/http.ex

```diff
@@ -0,0 +1,650 @@
+defmodule Snowflex.Transport.Http do
+  @moduledoc """
+  REST API transport implementation for Snowflake.
+  See: https://docs.snowflake.com/en/developer-guide/sql-api/reference
+
+  ## Configuration Options
```

Comment:
It would be nice to reference these config options in the `README` for easier discoverability. If you're unfamiliar with the project's code structure, it's not clear how to find these otherwise.

Reply:
The generated hexdocs provide a reference to this.  Do you feel like that is enough?

<img width="938" height="687" alt="Screenshot 2025-07-31 at 12 42 23 PM" src="https://github.com/user-attachments/assets/81411e7e-5257-4b6e-a932-42bc96f3d5a8" />

Reply:
Missed that, that's perfect!